README.rst

```diff
@@ -142,6 +142,16 @@ Given that, you can run Cloud Custodian with::
   # Run the policy
   $ custodian run -s out policy.yml
 
+You can run it with Docker as well
+
+  # Download the image
+  $ docker pull cloudcustodian/c7n
+
+  # Run the policy
+  $ docker run -it \
+      -v $(pwd)/policy.yml:/src/policy.yml \
+      --env-file <(env | grep "^AWS") \
+      cloudcustodian/c7n run -s out /src/policies/policy.yml
```

Comment:
probably want a volume -v for the output directory as well. we also may need to cover on examples for gcp/azure,  I  can add those after the fact. its mostly just the specific credential file / env var needed for those providers.

Reply:
Good idea. I added `-v` to it.

Reply:
The policy was associated to AWS so I figured making one specific to AWS was okay but yeah examples for GCP and Azure would be nice too